libpod/image/image.go

```diff
@@ -14,6 +14,7 @@ import (
 	"syscall"
 	"time"
 
+	commonimage "github.com/containers/common/pkg/image"
```

Comment:
Is @mtrmac happy with this package name?  I thought he raised concerns over in common (https://github.com/containers/common/pull/229) and I'd like to settle that before we move forward with this PR.

Reply:
That really depends on what else is planned to go into that package.

The retry code is somewhat image-independent (but not entirely, it does check for registry errors, and drag in a registry subpackage) and could be used for other network operations, so I don’t think it’s a good name for a package containing _only_ this code. A larger package containing several more image-related utilities might make more sense, OTOH it’s currently not clear to me that that would have to be a single package versus a set of utility subpackages (from which only a subset could be used to minimize dependencies in consumers).

Reply:
This package plans to merge the work from  @ParkerVR pulling of shortnames .

Reply:
Yeah — is there value in having both in a single subpackage? Skopeo won’t use the short name part, at least.

(I don’t really want to make a big deal out of this, the c/common has no API guarantee, it’s not _that much_ code I expect, and the repos are tightly coupled enough that we can rearrange any time. I guess dragging in the full containers.conf config file, or some user-observable non-optional initialization behavior, into Skopeo would be the point I’d start feeling stronger about this.)

Reply:
I don't object to having them both in the same package, but I do object to the name. Importing both `github.com/containers/libpod/v2/libpod/image` and `github.com/containers/common/pkg/image` in the same file is horribly confusing and renaming one at import time does not make things much better.

Reply:
thanks @mtrmac . I moved it to a single pkg. PTAL https://github.com/containers/common/pull/237

Reply:
I definitely do not like the name retry, it is too small, I am leaning torwards containers/common/pkg/pull.

Reply:
I agree with Dan - but do we want more organization with something like containers/common/pkg/image/pull ? This might give us more space for the future with less refactoring